.travis.yml

```diff
@@ -0,0 +1,38 @@
+sudo: required
+dist: trusty
+
+# install the pre-release chef-dk.  Use chef-stable-precise to install the stable release
+addons:
+  apt:
+    sources:
+      - chef-current-precise
+    packages:
+      - chefdk
+
+services: docker
+
+env:
+  matrix:
+  - INSTANCE=postgresql-ubuntu-1404
+  - INSTANCE=postgresql-ubuntu-1204
+  - INSTANCE=postgresql-centos-6
+  - INSTANCE=postgresql-centos-7
+
+# Don't `bundle install`
+install: echo "skip bundle install"
```

Comment:
Honestly i don't know anyone that actually uses it haha
AFAIK it was designed more for ease of getting started with chef, whereas most cookbooks do their own thing